src/ContainerInstance/ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -263,7 +263,7 @@ public override void ExecuteCmdlet()
                     {
                         throw new ArgumentException($"Invalid 'Command' parameter: {string.Join("; ", errors.Select(err => err.Message))}");
                     }
-                    creationParameter.ContainerCommand = tokens.Select(token => token.Text.Trim('\'', '"')).ToList();
+                    creationParameter.ContainerCommand = tokens.Select(token => token.Text.Trim('\'', '"')).Where(token => !string.IsNullOrEmpty(token)).ToList();
```

Comment:
Can you add a note under 'UpcomingRelease' here: https://github.com/Azure/azure-powershell/blob/master/src/ContainerInstance/ContainerInstance/ChangeLog.md#upcoming-release

Reply:
Done.